libs/domain/site/src/services/gender/default-gender.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Observable, of } from 'rxjs';
+import { Option } from '../../models';
+import { GenderService } from './gender.service';
+
+export class DefaultGenderService implements GenderService {
+  protected genders = [
+    { text: 'male', value: 'Male' },
+    { text: 'female', value: 'Female' },
+  ];
```

Comment:
Is there no Spryker API that exposes this list?

Reply:
No, there is no endpoint(s) that exposes the list of genders and salutations